Joining together CTRP data.Rmd

```diff
@@ -0,0 +1,304 @@
+---
+title: "Joining together CTRP data"
+output: github_document
+---
+
+## 1. Load packages
+```{r load packages}
+
+library(data.table)
+library(readr)
+library(dplyr)
+library(tidyverse)
+library(knitr)
+library(lubridate)
+
+```
+
+## 2. Load data
+```{r load data}
+
+setwd("~/Desktop/GitHub/Homework_5/HW05/CTRP_files")
+
+AUC_1 <- fread("AUC_1.csv")
+AUC_2 <- fread("AUC_2.csv")
+experiment_info <- fread("Experiment_info.csv")
+cancer_cell_line <- fread("cancer_cell_line_info.csv")
+compound <- fread("compound_info.csv")
+  
+```
+
+## 3. Tidy the data frames
+
+### 3a. AUC_1
+```{r AUC_1}
+
+# Rename the columns in AUC_1 file
```

Comment:
It's a great idea to rename the columns so that they're the same in each data frame. This definitely helps when trying to join data frames together by column names.